ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -216,23 +217,143 @@ public int getMaxTrailingPeers() {
   public static class Builder {
     private SyncMode syncMode = SyncMode.FULL;
     private Range<Long> blockPropagationRange = Range.closed(-10L, 30L);
+
+    @CommandLine.Option(
+        names = "--Xsynchronizer-downloader-change-target-threshold-by-height",
+        hidden = true,
+        defaultValue = "20",
+        paramLabel = "<LONG>",
+        description =
+            "Minimum height difference before switching fast sync download peers (default: ${DEFAULT-VALUE})")
     private long downloaderChangeTargetThresholdByHeight = 20L;
+
+    //  @CommandLine.Option(
+    //      names = "--Xsynchronizer-downloader-change-target-threshold-by-td",
+    //      hidden = true,
+    //      defaultValue = "1000000000",
+    //      paramLabel = "<UINT256>",
+    //      description =
+    //          "Minimum total difficulty difference before switching fast sync download peers
+    // (default: ${DEFAULT-VALUE})")
```

Comment:
nit: Probably should just remove the annotation instead of commenting it out.

Reply:
I added support for UInt256 and restored this annotation.